test/cctest/test-macro-assembler-riscv64.cc

```diff
@@ -129,9 +129,10 @@ TEST(LoadAddress) {
   __ jr(ra);
   __ nop();
   __ bind(&skip);
-  __ li(a4, Operand(masm->jump_address(&to_jump)), ADDRESS_LOAD);
+  __ LoadAddress(a4, &to_jump);
+  // __ li(a4, Operand(masm->jump_address(&to_jump)), ADDRESS_LOAD);
```

Comment:
Let's change this back to using the `li` since your modification catches that case properly.